Sources/Classes/Public/RSEventRepository.h

```diff
@@ -59,6 +61,10 @@ NS_ASSUME_NONNULL_BEGIN
 - (RSConfig* _Nullable) getConfig;
 - (void)_applicationDidFinishLaunchingWithOptions:(NSDictionary *)launchOptions;
 
+- (void)applyIntegrations:(RSMessage *)message withDefaultOption:(RSOption *)defaultOption;
+- (RSMessage *)applyConsents:(RSMessage *)message withConsentFilter:(RSConsentFilter *)consentFilter;
+- (void)applySession:(RSMessage *)message withUserSession:(RSUserSession *)userSession andRudderConfig:(RSConfig *)rudderConfig;
```

Comment:
Add a comment for `applyIntegrations` and `applySession` methods to signify that they are visible only for testing.

Reply:
Added.